README.rst

```diff
@@ -12,7 +12,7 @@ federated-content-connector
 |license-badge| |status-badge|
 
 Purpose
-*******
+-------
 
 One-line description for README and other doc files.
```

Comment:
Can we add a description " Plugin for integration layer of externally hosted content"